pom.xml

```diff
@@ -122,6 +122,19 @@
     </build>
     <dependencyManagement>
         <dependencies>
+        
+        <!-- AWS SDK Core -->
+  <dependency>
+    <groupId>software.amazon.awssdk</groupId>
+    <artifactId>bedrockruntime</artifactId>
+    <version>2.31.73</version>
+  </dependency>
+<dependency>
+  <groupId>software.amazon.awssdk</groupId>
+  <artifactId>url-connection-client</artifactId>
+   <version>2.31.73</version>
+</dependency>
+        
```

Comment:
This whole block is not required because we already import the BOM.  Please remove these two under dependencyManagement.